src/base/static/components/organisms/main-map.js

```diff
@@ -231,6 +232,10 @@ class MainMap extends Component {
         }
       });
     }
+
+    requestAnimationFrame(() => {
```

Comment:
Still not completely sure with this `requestAnimationFrame` is needed, but without it the map container will be sized incorrectly on an initial direct route to a Place.